networktest.go

```diff
@@ -53,6 +54,11 @@ var (
 	defaultClientPort = 19556
 
 	harnessNetParams = &chaincfg.SimNetParams
+
+	// logOutput is a flag that can be set to append the output from the
+	// seed nodes to log files.
+	logOutput = flag.Bool("logoutput", false,
```

Comment:
This will still _unconditionally_ log the output of _all_ nodes. Instead, here's a suggestion: change the type to be a `flat.Int` instead. Then the default value should be set to `-1`, which indicates don't log any node's output. Setting the flag to a value indicates that only that particular node should be logged.